### ISSUE TITLE
Add support for external testnets

### ISSUE THREAD

**Issue by jim-toth** [ArweaveTeam/testweave-sdk#3]
Hi,

I've recently started a project using Arweave and long story short I'm using a cloud container to host my testnet rather than run it locally.  I've got the testnet running and responding to external requests, and when I don't initialize TestWeave everything works fine.

I noticed an uncaught error where the http response was undefined.  I was able to trace this down to `TestWeaveTransactionsManager` where it's explicitly overriding the `baseURL` of the request to `localhost`

https://github.com/ArweaveTeam/testweave-sdk/blob/main/src/classes/class.testweave-transactions-manager.ts#L69
```ts
request.defaults.baseURL = 'http://localhost';
```

I imagine the same issue would arise later on in the `getPost()` method on this line:
https://github.com/ArweaveTeam/testweave-sdk/blob/main/src/classes/class.testweave-transactions-manager.ts#L69
```ts
if (endpoint === 'graphql') {
  request.defaults.baseURL = 'http://localhost';
}
```

Would it be possible to use the same baseURL building logic from ArweaveJS to allow for external testnets based on the config passed to the Arweave instance?  e.g.

```ts
const config = this._arweave.api.config
request.defaults.baseURL = `${config.protocol}://${config.host}:${config.port}`
```

**Comment by akangaziz** [comment]
no response (yet)?

**Comment by wshino** [comment]
I have encountered the same problem. node is running on GKE. I would like to support this change.

**Comment by pnutmath** [comment]
Just commenting/removing those line should work! @jim-toth

**Comment by jim-toth** [comment]
> Just commenting/removing those line should work! @jim-toth

definitely!  I haven't looked at the source since I opened this issue but iirc there are other parts in the code that would prevent this from working outside of localhost

**Comment by pnutmath** [comment]
Hi @jim-toth, I have setup this on EC2. It works for me till now apart from balance transfer, it keeps on waiting state!

Created fork and made some smaller changes

`"testweave-sdk": "github:pnutmath/testweave-sdk#develop"`

**Comment by xudexa** [comment]
Hello,

I have a problem with the baseURL=localhost.
Would it be possible to merge the branch.

Thank you very much

**Comment by saadtroon** [comment]
When will this branch be merged?
